server/utils.py

```diff
@@ -27,7 +28,7 @@ class RedisSettings(BaseSettings):
     ssl_certfile: Optional[str] = None
     ssl_keyfile: Optional[str] = None
 
-    model_config = SettingsConfigDict(
+    model_config = ConfigDict(
```

Comment:
can you explain the changes?

Reply:
Yep, the subclasses inheriting from BaseSettings meant they would try to look for env vars to pick up on their own independently, now they don't and only Settings checks for env vars